File: Practice problem/N numbers with an interval of X.cs
// 연습문제 x만큼 간격이 있는 n개의 숫자 Level 1

public class Solution {
    public long[] solution(int x, int n) {
        long[] answer = new long[n];
        for(int i = 1; i <= n; i++){ answer[i-1] = (long)x*i; }
        return answer;
    }
}

File: Practice problem/N numbers with an interval of X/N numbers with an interval of X.cpp
// 연습문제 x만큼 간격이 있는 n개의 숫자 Level 1

#include <string>
#include <vector>

using namespace std;

vector<long long> solution(int x, int n) {
    vector<long long> answer;
    for(int i = 1; i <= n; i++){ answer.push_back(x*i); }
    return answer;
}
